cli/tests/integrations/test_task.py

```diff
@@ -257,6 +257,30 @@ def test_ls_bad_path():
         returncode=1)
 
 
+def test_ls_completed():
+    # create a completed task
+    with app(SLEEP_COMPLETED, 'test-app-completed'):
+        # get its task id
+        task_id_completed = _get_completed_task_id()
+
```

Comment:
@surdy since we are creating `test-app-completed` here, can we remove it from the setup module?